npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker/version_resolver.rb

```diff
@@ -519,7 +521,7 @@ def run_yarn_berry_checker(path:, version:)
           end
         end
 
-        def run_yarn_checker(path:, version:)
+        def run_yarn_classic_checker(path:, version:)
```

Comment:
Is `yarn_classic` Yarn v1? If so, a code comment might be helpful context for those less familiar with the ecosystem...

Reply:
Yes, it is. Since Yarn v3 is referred to as yarn berry, I thought I'd use yarn classic here. And also because I'm not sure where yarn v2 falls, or whether it exists at all 😆. I'm still figuring out yarn versioning but I read "classic" a few times when meaning "yarn before berry", that's why I used it here because it seemed a common terminology for Yarn people.